label_studio/io_storages/base_models.py

```diff
@@ -665,10 +698,13 @@ def sync(self):
             ):
                 if not self.info_set_queued():
                     return
-                sync_job = queue.enqueue(
+                # Use start_job_async_or_sync to automatically capture and restore CurrentContext
+                # This ensures user_id, organization_id, and request_id are available in the worker
+                sync_job = start_job_async_or_sync(
                     import_sync_background,
                     self.__class__,
                     self.id,
+                    queue_name='low',
```

Comment:
can get rid of `queue = django_rq.get_queue('low')` above now to avoid confusion